### PR TITLE
User Description 

### PR DESCRIPTION
**Pull request by montitawi**
First Pull Request

#1 resolved

### CODE REVIEW

File: PARTICIPANTS/montitawi.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (MJ10) [inline PARTICIPANTS/montitawi.txt]: Please move this file to the `Participants` directory

**Comment by MJ10** [comment]
@montitawi Any updates or should we close this and mark as spam?

**Comment by montitawi** [comment]
Sorry I will update today but now I can’t update.

**Comment by salman-bhai** [comment]
Please try to resolve whenever you can!

**Comment by montitawi** [comment]
I update now. I hope it correct.
I'm so sorry.

**Comment by MJ10** [comment]
@montitawi It's okay. You haven't put the file in the `Participants` directory though. Just deleted it. Please add the file in the `Participants` directory and you'll be done with this.

**Pull request merged**